pandas/core/window/numba_.py

```diff
@@ -220,3 +220,20 @@ def roll_table(
         return result
 
     return roll_table
+
+
+# This function will no longer be needed once numba supports
+# axis for all np.nan* agg functions
+# https://github.com/numba/numba/issues/1269
+def generate_manual_numpy_nan_agg_with_axis(nan_func):
+    numba = import_optional_dependency("numba")
```

Comment:
you need the import here?

Reply:
Yeah it's the pattern I've been using in other functions in this file since numba is an optional dependency